test/indexer/test_anvil.py

```diff
@@ -109,6 +119,17 @@ def test_indexing(self):
                     finally:
                         self.index_service.delete_indices(self.catalog)
 
+    def test_fetch_bundle(self):
+        canned_bundle = self._load_canned_bundle(self.bundle)
+        assert isinstance(canned_bundle, TDRAnvilBundle)
+        self._make_mock_tdr_tables(self.bundle)
+        plugin = self.plugin_for_source_spec(canned_bundle.fqid.source.spec)
+        bundle = plugin.fetch_bundle(self.bundle)
+        assert isinstance(bundle, TDRAnvilBundle)
+        self.assertEqual(canned_bundle.fqid, bundle.fqid)
```

Comment:
These are fairly weak assertions. Can we compare the result to the "intermediate" can?

Reply:
Please disregard.